File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    longestConsecutive(nums) {
        let longest = new Set(nums)
        let res = 0

        for(let num of nums){
            let curr = num, streak = 0
            while(longest.has(curr)){
                streak++
                curr++
            }
            res = Math.max(streak, res)
        }
        return res
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        count = 0
        maxnum = 0
        for i in range(len(nums)):
            if nums[i] == 1:
                count += 1
                maxnum = max(maxnum, count)
            else:
                count = 0
        
        return maxnum